package.json

```diff
@@ -1,31 +1,38 @@
 {
-    "devDependencies": {
-      "husky": "^0.14.3",
-      "codecov": "^2.2.0",
-      "jest": "^21.2.1",
-      "lerna": "^2.3.1",
-      "lint-staged": "^4.2.3",
-      "prettier-eslint-cli": "^4.4.0"
-    },
-    "scripts": {
-      "postinstall": "lerna bootstrap",
-      "coverage": "codecov",
-      "clean": "lerna clean",
-      "updated": "lerna updated",
-      "test": "jest",
-      "publish": "lerna publish --exact",
-      "precommit": "lint-staged"
-    },
-    "lint-staged": {
-      "*.js": [
-        "prettier-eslint --write",
-        "jest --findRelatedTests",
-        "git add"
-      ]
-    },
-    "jest": {
-      "rootDir": "packages",
-      "coverageDirectory": "./coverage/",
-      "collectCoverage": true
-    }
-  }
\ No newline at end of file
+  "scripts": {
+    "postinstall": "lerna bootstrap",
+    "coverage": "codecov",
+    "clean": "lerna clean",
+    "updated": "lerna updated",
+    "test": "jest",
+    "publish": "lerna publish --exact",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier-eslint --write",
+      "jest --findRelatedTests",
+      "git add"
+    ]
+  },
+  "jest": {
+    "rootDir": "packages",
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true
+  },
+  "devDependencies": {
+    "codecov": "^2.2.0",
+    "husky": "^0.14.3",
+    "jest": "^21.2.1",
+    "lerna": "^2.3.1",
+    "lint-staged": "^4.2.3",
+    "prettier-eslint-cli": "^4.4.0",
+    "async": "^2.5.0",
```

Comment:
I think there is no need anymore for those* here they are now correctly defined as dependencies in the respective packages (ie: `packages/oc-s3-storage-adapter/package.json`).


- async
- aws-sdk
- fs-extra
- lodash
- nice-cache
- node-dir
- stringformat